schema.sql

```diff
@@ -11,4 +11,47 @@ CREATE TABLE animals (
   
   -- Adding new table header
 ALTER TABLE animals
-ADD COLUMN species VARCHAR(100);
\ No newline at end of file
+ADD COLUMN species VARCHAR(100);
+
+--  Create a table named owners
+CREATE TABLE owners(
+id BIGSERIAL NOT NULL PRIMARY KEY ,
+full_name VARCHAR (100) NOT NULL,
+age INT NOT NULL
+);
+
+
+
+--  Create a table named species
+CREATE TABLE species (
+id BIGSERIAL NOT NULL PRIMARY KEY ,
+name VARCHAR (100) NOT NULL
+ );
```

Comment:
- [ ] You missed the first requirement which was the creation of the `vets` table. Kindly find below the snapshot of the full requirements for that table which needs to be implemented 👍 

### Missing table and requirements:
![image](https://github.com/emmanuelsteven/vet_clinic/assets/50990925/30c4901c-be9f-4e34-b3f0-ad7a54550f29)